components/authz-service/storage/postgres/datamigration/sql/07_update_v2_roles.up.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+
+UPDATE iam_roles
+    SET
+        actions = actions || '{applications:*:list, applications:*:get}'
+    WHERE
+        id = 'viewer';
+
+UPDATE iam_roles
+    SET
+        actions = actions || '{applications:*}'
+    WHERE
+        id = 'editor';
+END;
```

Comment:
What is this `END` doing that's distinct from `COMMIT`?

Reply:
probably i just wanted commit and end snuck in there. Commit will commit a transaction, end just delineates a block of code, like if it were in a loop. I think maybe I copied this structure from a loop somewhere?